File: NekretnineFrontend/src/app/grad/grad.ts
export class Grad{
  public gradId : number;
  public  drzavaID : number;
  public naziv : string;
  public izmjenaGrada : boolean;
  constructor(gradId : number, drzavaID : number, naziv : string, izmjenaGrada : boolean)
  {
    this.gradId = gradId;
    this.drzavaID = drzavaID;
    this.naziv = naziv;
    this.izmjenaGrada = izmjenaGrada;
  }

}

File: NekretnineAPI/Database/Grad.cs
using System;
using System.Collections.Generic;

namespace Database
{
    public partial class Grad
    {
        public Grad()
        {
            Lokacijes = new HashSet<Lokacije>();
        }

        public int GradId { get; set; }
        public string? Naziv { get; set; }
        public int DrzavaId { get; set; }

        public virtual Drzave Drzava { get; set; } = null!;
        public virtual ICollection<Lokacije> Lokacijes { get; set; }
    }
}
